.vaunt/config.yaml

```diff
@@ -0,0 +1,159 @@
+version: 0.0.1
+actions:
+    - action:
+          type: point
+          name: docs_dash_pr_submitted
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & labels in {'2024-tidb-docs-dash','contribution'}"
+                    value: 4
+    - action:
+          type: point
+          name: docs_dash_pr_merged
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-19 20:00:00
```

Comment:
Can you updated the merged at end date to `2024-01-26 20:00:00` as we decided to give extra time for reviews & merges?